.travis.yml

```diff
@@ -1,41 +1,23 @@
 language: php
 
-php: [5.6, 7.0, 7.1, 7.2, 7.3, 7.4]
-
-sudo: false
+php: [7.2, 7.3, 7.4]
 
 cache:
   directories:
     - $HOME/.composer/cache
 
 matrix:
   include:
-    - php: 5.3
-      dist: precise
-      env: PHP_VERSION=53
-    - php: 5.4
-      dist: trusty
-      env: PHP_VERSION=54+
-    - php: 5.5
-      dist: trusty
-      env: PHP_VERSION=54+
-    - php: 5.6
-      env: DEPENDENCIES='low'
-      env: PHP_VERSION=54+
-    - php: 5.6
+    - php: 7.2
       env: SKIP_INTEROP='~@interop&&'
-    - php: 5.6
-      env: SYMFONY_VERSION='2.7.*'
-    - php: 5.6
-      env: SYMFONY_VERSION='2.8.*'
     - php: 7.2
-      env: SYMFONY_VERSION='^3.4'
-    - php: 7.3
-      env: SYMFONY_VERSION='^3.4'
-    - php: 7.3
-      env: SYMFONY_VERSION='^4.3'
+      env: DEPENDENCIES='low'
+    - php: 7.2
+      env: SYMFONY_VERSION='^4.4'
     - php: 7.3
-      env: SYMFONY_VERSION='^5.0'
+      env: SYMFONY_VERSION='^4.4'
```

Comment:
Do we actually need 3 jobs running against the Symfony 4.4 LTS ?

Reply:
I guess not really, don't have a better solution for now though - we could iterate over it later.

Reply:
I mean, just remove some of these jobs, keeping only one testing against the 4.4 LTS. We already have earlier jobs testing each PHP version.

Reply:
Left PHP 7.4 & Symfony 4.4 only.